tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -108,23 +108,23 @@ steps:
                   exit 1
                 fi
             else
-              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+              feedName=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
 
-              if [[ $release == "release" && $feedId == "build" ]]; then
+              if [[ "$(release)" == "release" && $feedName == "build" ]]; then
```

Comment:
Probably not for this PR, but this sounds like we don't promote things published to the `test` feed, or directly to the `dev` feed? So could we eventually run into the same problem with those? Admittedly, they probably see much less volume than `build`.